pkg/driver/node_server.go

```diff
@@ -1656,6 +1656,11 @@ func (driver *Driver) NodeGetVolumeStats(ctx context.Context, in *csi.NodeGetVol
 	log.Trace(">>>>> NodeGetVolumeStats")
 	defer log.Trace("<<<<< NodeGetVolumeStats")
 
+	volumeID := in.GetVolumeId()
+	if volumeID == "" {
+		return nil, status.Errorf(codes.InvalidArgument, "volume id %s is empty", volumeID)
```

Comment:
we didn't need the volumeID in the message if its empty

Reply:
fixed